packages/renderer/src/lib/flows/types/input-field.ts

```diff
@@ -15,7 +15,11 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
-export interface ChunkProviderInfo {
-  id: string;
+
+export interface InputField {
```

Comment:
suggestion(non-blocking): we could use zod to build this interface and so we could validate the form, I will submit a suggestion PR